nhs_context_logging/logger.py

```diff
@@ -385,7 +385,7 @@ def get_method_name(func, *args, **kwargs):
         method_name = f"{cls.__name__}.{func.__name__}"
     else:
         method_name = func.__name__
-    return method_name
+    return f"{func.__module__}.{method_name}"
```

Comment:
this needs to be an optional (default=False) behaviour .. this would break a lot of the existing MESH dashboards tests, etc...

Reply:
good to know! that's done now